src/components/Tree.tsx

```diff
@@ -329,16 +329,28 @@ export class Tree extends React.PureComponent<TreeProps, {}> {
      * Generates the css classes for indenting the nodes.
      *
      * @param {number} depth Max depth of the tree. This is how many classes will be generated.
-     * @returns {string} CSS: .indent-X{padding-left:X*15px}
+     * @returns {string} CSS for .indent-X with and without .no-open-button
      */
-    private static generateIndentCSS(depth: number): string {
+    private static generateIndentCSS(depth: number, props: any): string {
```

Comment:
Can we have the concrete function passed here instead of props? I know that Typescript is not that convenient, but I would like to keep params as specific as possible. (And please add the comment for the new param, it is used in the automatic documentation generation.)

Reply:
Sure, added, and updated types :)